qemu/tests/emulate_vf_reboot.py

```diff
@@ -0,0 +1,26 @@
+from virttest import error_context
+
+from provider import create_emulate_vf
```

Comment:
I am thinking about the module name, if it is only to create guest vfs, that's not enough to be a module. Also, the module name is `create_emulate_vf`, which means it only for creating VFs.

Should make sure it is only be used for VF? Or will add other functions to handle others in the future, like PF, or bind drivers, and then can define a suitable name.